Gemfile

```diff
@@ -13,3 +13,5 @@ gemspec
 # To use a debugger
 # gem 'byebug', group: [:development, :test]
 
+
+gem 'rspec-html-matchers', git: 'https://github.com/ministryofjustice/rspec-html-matchers.git', branch: 'fix-deep-nesting'
```

Comment:
Once the rspec-html-matcher PR is merged we should remove the reference to the branch. Ideally, we should get push this up. We might want to mention in the forked repo that its just being used for our form builder so if someone looks at https://github.com/ministryofjustice/rspec-html-matchers.git and wonders what it's being used for and simply deletes the repo then it would break the tests.